src/services/analytics/events/txList.ts

```diff
@@ -8,7 +8,7 @@ export const TX_LIST_EVENTS = {
     action: 'Queued transactions',
     category: TX_LIST_CATEGORY,
   },
-  // TODO: label: 'Edit' does not exist in web-core
+  // 'Edit' label does not exist in web-core
```

Comment:
Should we remove this comment? I don't really understand it tbh.
AFAIK this is the Action in the extended menu of a transfer transaction to add a receiver to the addressbook. 
What does this have to do with an "Edit label"? 
Or does the comment not relate to the `ADDRESS_BOOK` Event?

Reply:
As per our discussion, we've simplified this to match that of the transaction list context menu.